lib/flay.rb

```diff
@@ -548,6 +611,21 @@ def report io = $stdout
     end
   end
 
+  ##
+  # Output the report. Duh.
+
+  def report io = $stdout
+    only = option[:only]
+
+    data = analyze only
+
+    if option[:report]
+      report_json(io,data)
+    else
+      report_io(io,data)
```

Comment:
not sure what you mean. Didn't see anything about conventions, and I'm not used to ruby and to its best code style practices. If you can specify the problem I will fix it in no time